test/serializers/associations_test.rb

```diff
@@ -261,6 +261,30 @@ def false
 
           assert_equal(expected, hash)
         end
+
+        def test_associations_yield_reflections
+          model = ModelWithAssociations::Model.new(
+            id: 1337,
+            title: 'New Post',
+            blog: nil,
+            body: 'Body',
+            comments: (0..50).map { |i| ModelWithAssociations::Comment.new(id: i, body: 'ZOMG A COMMENT') },
```

Comment:
Thanks, will look into it in half an hour 👌

On Monday, 18 April 2016, Benjamin Fleischer <notifications@github.com>
wrote:

> In test/serializers/associations_test.rb
> <https://github.com/rails-api/active_model_serializers/pull/1690#discussion_r60128189>
> :
>
> > @@ -261,6 +261,30 @@ def false
> >
> >            assert_equal(expected, hash)
> >          end
> > +
> > +        def test_associations_yield_reflections
> > +          model = ModelWithAssociations::Model.new(
> > +            id: 1337,
> > +            title: 'New Post',
> > +            blog: nil,
> > +            body: 'Body',
> > +            comments: (0..50).map { |i| ModelWithAssociations::Comment.new(id: i, body: 'ZOMG A COMMENT') },
>
> not necessary, just was from my original work. Should be ✂️
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1690/files/b3518271c473ef14e73463cd5e93897a3d8e3785#r60128189>
>


-- 
Lucas Hosseini
lucas.hosseini@gmail.com